### PR TITLE
List aas descriptors endpoint

### PR DESCRIPTION
**Pull request by akiskips**
- Added a new route GET /AASDescriptors to the Endpoint Registry that returns a list of all AASDescriptors stored in the registry.
- Added tests for this
- Updated a the GET /AASDescriptor/aasId implementation to spare a db call

### CODE REVIEW

**Comment by Manu3756** [comment]
please add doc in README or Postman

**Comment by akiskips** [comment]
> please add doc in README or Postman

Updated the Readme, I will add a request to the postman collection after the TLS branch gets merged to avoid merge conflicts

**Comment by Manu3756** [comment]
Postman still missing.

File: docs/postman/AAS-Service.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1127 @@
+{
```
Comment (etoster) [inline docs/postman/AAS-Service.postman_collection.json:1]: Why is this a new separate Postman Collection file?

Comment (akiskips) [reply docs/postman/AAS-Service.postman_collection.json]: Good point... Seems that was exported with a different file name
I pushed an update

**Pull request merged**